src/absil/ilwrite.fs

```diff
@@ -222,7 +222,6 @@ type PdbData =
 //---------------------------------------------------------------------
 
 let WritePdbInfo fixupOverlappingSequencePoints showTimes f fpdb info = 
-    (try FileSystem.FileDelete fpdb with _ -> ())
```

Comment:
removed because useless, the file is now removed in `serializeToFile` (the same for the .mdb)